src/Avatar.ts

```diff
@@ -24,16 +24,19 @@ import DMRoomMap from "./utils/DMRoomMap";
 import { mediaFromMxc } from "./customisations/Media";
 import { isLocalRoom } from "./utils/localRoom/isLocalRoom";
 
+const DEFAULT_COLORS: Readonly<string[]> = ["#0DBD8B", "#368bd6", "#ac3ba8"];
```

Comment:
Where did these colours come from?

Reply:
https://github.com/matrix-org/matrix-react-sdk/blob/a69b805156126708c78e2a0ed7c5fab3919a7f82/src/Avatar.ts#L91

They were within `defaultAvatarUrlForString`, I just extracted them to a file-level constant.